src-cpp/QueueImpl.cpp

```diff
@@ -35,3 +35,22 @@ RdKafka::Queue *RdKafka::Queue::create (Handle *base) {
   queueimpl->queue_ = rd_kafka_queue_new(dynamic_cast<HandleImpl*>(base)->rk_);
   return queueimpl;
 }
+
+RdKafka::ErrorCode
+RdKafka::QueueImpl::forward (Queue *queue) {
+  if (!queue)
+    rd_kafka_queue_forward(queue_, NULL);
```

Comment:
this scope should probably return when done since the sub sequent code expects queue to be non-NULL